packages/autocomplete-core/package.json

```diff
@@ -6,18 +6,32 @@
   "source": "index.tsx",
   "types": "dist/es/index.d.ts",
   "module": "dist/es/index.js",
-  "main": "dist/umd/autocomplete-core.js",
-  "umd:main": "dist/umd/autocomplete-core.js",
-  "unpkg": "dist/umd/autocomplete-core.js",
-  "jsdelivr": "dist/umd/autocomplete-core.js",
+  "main": "dist/umd/index.js",
+  "umd:main": "dist/umd/index.js",
+  "unpkg": "dist/umd/index.js",
+  "jsdelivr": "dist/umd/index.js",
   "homepage": "https://github.com/algolia/autocomplete.js",
   "repository": "algolia/autocomplete.js",
+  "scripts": {
+    "build": "rm -rf ./dist && rollup --config && yarn build:types",
+    "build:types": "tsc -p tsconfig.declaration.json --outDir ./dist/es; echo ''"
```

Comment:
What is `; echo ''` for?

Reply:
It failed because of `__DEV__`.

Now it's fixed: https://github.com/francoischalifour/autocomplete.js/pull/26/commits/34d4e6bd7b30770d0f6e1f9bc3fc719ed7be97a2